MapboxNavigation/Bundle.swift

```diff
@@ -1,18 +1,29 @@
 import Foundation
 
 extension Bundle {
-    class var navigationUI: Bundle {
-        get { return Bundle(for: NavigationViewController.self) }
-    }
     
-    var backgroundModeLocationSupported: Bool {
-        get {
+    struct BackgroundModes {
+        var location: Bool {
+            return backgroundModesContains("location")
+        }
+        
+        var audio: Bool {
+            return backgroundModesContains("audio")
+        }
+        
+        fileprivate func backgroundModesContains(_ key: String) -> Bool {
```

Comment:
We could avoid this duplication by sharing Bundle.swift between the MapboxCoreNavigation and MapboxNavigation modules. (We’d have to rename this file to avoid a build error.)